webrtc-stats.html

```diff
@@ -1256,6 +1258,39 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>estimatedPlayoutTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for remote sources. This is the estimated playout time of this track.
+                  The playout time is the NTP timestamp of the last played out audio sample or video
+                  frame that has a known timestamp, extrapolated with the time elapsed since it was
+                  played out. This is the "current time" of the track and can be present even if
```

Comment:
Relative to the NTP clock of the sender sounds to me like if sender NTP time is 100 and rtt causes playout time to be at 90 this would be -10. I changed it to "This is the "current time" of the track in NTP clock time of the sender" to refer to the clock in terms of time units. I hope that wording makes sense.